content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -460,14 +460,22 @@ An example flow:
       The containers in the Pod receive the TERM signal at different times and in an arbitrary
       order. If the order of shutdowns matters, consider using a `preStop` hook to synchronize.
       {{< /note >}}
-1. At the same time as the kubelet is starting graceful shutdown, the control plane removes that
-   shutting-down Pod from Endpoints (and, if enabled, EndpointSlice) objects where these represent
+
+1. At the same time as the kubelet is starting graceful shutdown, the control plane evaluates whether
+   shutting-down Pod needs to be removed from Endpoints (and, if enabled, EndpointSlice) objects where these represent
```

Comment:
Hi @SergeyKanzhelev .
Is a 'shutting-down' Pod the same as a terminating Pod?

Possible edit:
At the same time as the kubelet is starting graceful shutdown, the control plane evaluates whether a terminating Pod needs to be removed from any Endpoints and if enabled, EndpointSlice objects. 

Possible edit:
Pods that shut down slowly should not continue to serve traffic and instead terminate the connection.

Reply:
applied first and mentioned "regular traffic" in second edit to match the last sentence about load balancers directing regular traffic. I hope it clarifies things